examples/hello_services/ios/HelloServices/AppDelegate.m

```diff
@@ -0,0 +1,28 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "AppDelegate.h"
+
+#import <Flutter/Flutter.h>
+#import "LocationProvider.h"
+
+@interface AppDelegate ()
+
+@end
+
+@implementation AppDelegate
+
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    FlutterDartProject* project = [[FlutterDartProject alloc] initFromDefaultSourceForConfiguration];
```

Comment:
The following items are leaked unless we have ARC turned on in this project:

* `FlutterDartProject`
* `UIWindow`
* `FlutterViewController`
* `LocationProvider`

We generally do not use ARC in any of our projects. If its an example, Xcode allows migration a project from ARC to non-ARC.

If we are using ARC, how is `LocationProvider` not being collected at the end of this scope? The memory management seems non-idiomatic.